spyder/config/base.py

```diff
@@ -34,9 +34,9 @@
 # SPYDER_DEV is (and *only* has to be) set in bootstrap.py
 DEV = os.environ.get('SPYDER_DEV')
 
-# For testing purposes
-# SPYDER_TEST can be set using the --test option of bootstrap.py
-TEST = os.environ.get('SPYDER_TEST')
+# Make Spyder use a temp clean configuration directory for testing purposes
+# SPYDER_CLEAN can be set using the --clean option of bootstrap.py
+CLEAN = os.environ.get('SPYDER_CLEAN')

```

Comment:
This should be

    CLEAN_CONFIG = os.environ.get('SPYDER_CLEAN_CONFIG')

Reply:
Changed to ``SAFE_MODE``/``SPYDER_SAFE_MODE`` respectively.